f8a_report/unknown_deps_report_helper.py

```diff
@@ -32,8 +32,10 @@ def get_past_unknown_deps(self):
                 unknown_deps = result.get('stacks_summary', {}).get(eco, {}).\
                     get('unique_unknown_dependencies_with_frequency', {})
                 for k, v in unknown_deps.items():
-                    pkg, ver = k.split()[0], k.split()[1]
-                    deps.append({'name': pkg, 'version': ver})
+                    pkg_ver = k.split()
+                    if len(pkg_ver) == 2:
```

Comment:
@yzainee I think that the better solution will be to catch an IndexError as well as a generic Exception here. We can print a logline for such an erroneous pkg_version combo in the `raise` block. It will help us track those scenarios.

Reply:
changes done